packages/logger/src/correlationId.ts

```diff
@@ -0,0 +1,61 @@
+import type { JSONObject } from '@aws-lambda-powertools/commons/types';
+import { search as JMESPathSearch } from '@aws-lambda-powertools/jmespath';
+import { PowertoolsFunctions } from '@aws-lambda-powertools/jmespath/functions';
+
+/**
+ * This function is used to search for a correlation ID in the event data and is a wrapper
+ * around the JMESPath search function. It allows you to specify a JMESPath expression
+ * to extract the correlation ID from the event data.
+ * @param expression - The JMESPath expression to use for searching the correlation ID.
+ * @param data - The event data to search in.
+ * @returns - The correlation ID extracted from the event data.
```

Comment:
Despite VSCode suggesting them (I don't know how to turn them off either, we have removed all of them from the docs).

Reply:
do you mean the return statements or the params?